modules/tinymce/src/core/main/ts/commands/LineHeight.ts

```diff
@@ -17,7 +17,9 @@ export const lineHeightQuery = (editor: Editor): string => mapRange(editor, (elm
   const specifiedStyle = TransformFind.closest(elm, (elm) => Css.getRaw(elm, 'line-height'), Fun.curry(Compare.eq, root));
   const computedStyle = () => {
     // Css.get returns computed values (in px), and parseFloat will strip any non-number suffix
-    const lineHeight = parseFloat(Css.get(elm, 'line-height'));
+    const lineHeightRaw = parseFloat(Css.get(elm, 'line-height'));
+    // TINY-7895: Safari sometimes gives us 22.39999999999 and things like that
+    const lineHeight = parseFloat(lineHeightRaw.toPrecision(3));
```

Comment:
This isn't going to work in a number of cases, as take a slight deviation of the number you've given:
```
> (22.39899999999).toPrecision(3)
'22.4'
```
This instead probably needs to find up to 3 decimal places and round to that (e.g., not just rely on using only 3 values (I really do hate some JS apis, such as this). So in the above example, we probably want this to be `22.399` not `22.4`.

Note: I see I myself made this same flaw in `Transitions.ts` :facepalm: